cmd/inspect/main.go

```diff
@@ -26,10 +29,9 @@ func run() error {
 	if err != nil {
 		return err
 	}
-	producer := queue.ProvideProducer(cfg.RabbitMq.GetUrl())
-	inspector := pod.NewInspector(pc, cfg.DeployableNamespaces,
-		pod.NewTTLDestroyHandler(&producer),
-		pod.NewIDHandler(),
+	producer := rabbitmq.ProvideProducer(logger, cfg.RabbitMq.GetUrl())
```

Comment:
```suggestion
	producer := rabbitmq.NewProducer(logger, cfg.RabbitMq.GetUrl())
```

does not have to be now but at some point